Source/gui/SNTBinaryMessageWindowController.m

```diff
@@ -92,6 +92,9 @@ - (void)loadWindow {
     NSString *eventDetailText = [[SNTConfigurator configurator] eventDetailText];
     if (eventDetailText) {
       [self.openEventButton setTitle:eventDetailText];
+      // Require the button keyEquivalent set to be CMD + Return
+      [self.openEventButton setKeyEquivalent:@"\r"]; // Return Key
+      [self.openEventButton setKeyEquivalentModifierMask:NSCommandKeyMask]; // Command Key
```

Comment:
require cmd + enter from PR feedback

Reply:
Use `NSEventModifierFlagCommand`

Reply:
gotcha...lols, really moving the button backwards with that. for ref: https://developer.apple.com/documentation/appkit/nseventmodifierflags/nseventmodifierflagcommand.